src/feed/EventPublisher.js

```diff
@@ -28,6 +30,25 @@ export default function useEventPublisher() {
         return ev;
     }
 
+
+    function processMentions(ev, msg) {
+        const replaceHexKey = (match) => {
```

Comment:
Trying to remove hex keys i think, only `npub` / `note` keys should be extracted

Reply:
Leaving note references out for now, this PR only adds pubkey mentions with bech32 notation.